project_structure.txt

```diff
@@ -0,0 +1,101 @@
+.modin/
+   .config/
+   .distributed/
+   .pandas/
+   .test/
+   .rest files
+   .core/
+      .storage_formats/
```

Comment:
@devin-petersohn 

Doing the refactor one thought came in my mind. I see three options:

1. Leave as it is now
2. Put `storage_formats` along with `execution` in `dataframe`.
3. Move `execution` to the higher level to reside it along with `storage_formats` in `core`.

I am more inclined towards the third option. What do you think would be the best choice?

Reply:
3rd option is currently applied.

Reply:
Yes, the 3rd option makes the most sense to me